Src/PerformanceCollector/Shared/Implementation/WebAppPerformanceCollector/CacheHelper.cs

```diff
@@ -57,9 +57,9 @@ public int PerformanceCounterValue(string performanceCounterName, string json)
                 startingIndex++;
             }
 
-            if (!int.TryParse(valueString.ToString(), out value))
+            if (!long.TryParse(valueString.ToString(), out value))
             {
-                throw new System.InvalidCastException("The value of the counter cannot be converted to integer type.");
+                throw new System.InvalidCastException("The value of the counter cannot be converted to long type. Value:" + valueString);
```

Comment:
so what will happen here? Counter will be marked as bad and wouldn't be reported for 5 minutes? Or something else will happen?